EXT_mesh_features/ComplexTypes/tileset.json

```diff
@@ -0,0 +1,21 @@
+{
+  "asset" : {
+    "version" : "1.0"
+  },
+  "geometricError" : 2.0,
+  "root" : {
+    "content" : {
+      "uri" : "ComplexTypes.gltf"
+    },
+    "boundingVolume" : {
+      "box" : [ 
+        0.5, 0.0, 0.5,
+        0.5, 0.0, 0.0, 
+        0.0, 0.01, 0.0, 
+        0.0, 0.0, 0.5]
+    },
+    "geometricError" : 1.0
+  },
+  "extensionsUsed" : [ "3DTILES_content_gltf" ],
+  "extensionsRequired" : [ "3DTILES_content_gltf" ]
```

Comment:
While not strictly necessary, it would be good practice to list out the glTF extensions like in https://github.com/CesiumGS/3d-tiles/tree/main/extensions/3DTILES_content_gltf#extension-json